### PR TITLE
Sections view and Statistics view added

### PR DESCRIPTION
**Pull request by gwyndol1ne**

### CODE REVIEW

File: src/main/webapp/src/components/RuleViolationsHeader/RuleViolationsHeader.tsx [hunk 1]
```diff
@@ -37,14 +38,20 @@ const RuleViolationsHeader: React.FC<RuleViolationsHeaderProps> = ({ tabs }) =>
...
           <Tabs.List>{items}</Tabs.List>
 
           <Tabs.Panel value="Нарушения">
+
             <RuleViolationsContent />
           </Tabs.Panel>
-
+          <Tabs.Panel value="Статистика">
+            <Statistics/>
+        </Tabs.Panel>
```
Comment (Darderion) [inline src/main/webapp/src/components/RuleViolationsHeader/RuleViolationsHeader.tsx:51]: Inconsistent indentations

File: src/main/webapp/src/components/Sections/PDFLine.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import {Text} from '@mantine/core';
+
+interface PDFLinesProps {
+  lines: PDFLine[];
+}
+
+const PDFLines: React.FC<PDFLinesProps> = ({ lines }) => {
+  const getColorByArea = (area: string) => {
+    const colors = ['#FF7F50', '#FFD700', '#00FFFF', '#FF00FF', '#00FF00', '#800080', '#FFFF00', '#00CED1', '#FFA500', '#008000'];
+    let index = 0;
```
Comment (Darderion) [inline src/main/webapp/src/components/Sections/PDFLine.tsx:12]: Could replace a verbose switch case with an object representing a dictionary

File: src/main/webapp/src/components/Sections/PDFLine.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            }}
+          >
+            <Text align="center">{line.content}</Text>
+          </div>
+        ))}
+      </div>
+  );
+};
+
+export default PDFLines;
```
Comment (Darderion) [inline src/main/webapp/src/components/Sections/PDFLine.tsx:66]: Refrain from ignoring warning Github provides, such as "no newline character"

File: src/main/webapp/src/components/Statistics/Statistics.tsx [hunk 4]
```diff
@@ -1,92 +1,94 @@
...
-    axios.get<ApiResponse>(apiUrl+`/viewPDFStatistic?pdfName=${currentFileName}`, {
-      responseType: 'json',
-    })
+    axios.get(apiUrl+`/viewPDFStatistic?pdfName=${currentFileName}`)
       .then(res => {
-        const sortedWords: WordData[] = Object.entries(res.data.wordsStatistic.topKWords)
-          .sort((a, b) => b[1] - a[1])
-          .slice(0, 100)
-          .map(([word, count], index) => ({ number: index + 1, word, count }));
+        const sortedWords:any = Object.entries(res.data.wordsStatistic.topKWords)
```
Comment (Darderion) [inline src/main/webapp/src/components/Statistics/Statistics.tsx:20]: Use appropriate types

File: src/main/webapp/src/components/Statistics/Statistics.tsx [hunk 5]
```diff
@@ -1,92 +1,94 @@
...
+            .sort((a:any, b:any) => b[1] - a[1]) 
+            .slice(0, 100)
+            .map(([word, count], index) => ({ number: index + 1, word, count }));
         setWordData(sortedWords);
-        setWordCount(res.data.wordsStatistic.wordCount);
-
-        const pageResults: PageData[] = res.data.pageStatistic.sectionsStatistic.map(item => ({
+        setWordCount(res.data.wordsStatistic.wordCount);
```
Comment (Darderion) [inline src/main/webapp/src/components/Statistics/Statistics.tsx:25]: Refrain from using trailing whitespaces

**Review by Darderion — CHANGES_REQUESTED** [review]

File: src/main/webapp/src/components/RuleViolationsHeader/RuleViolationsHeader.tsx (hunk 1 above)
Comment (gwyndol1ne) [reply src/main/webapp/src/components/RuleViolationsHeader/RuleViolationsHeader.tsx]: .

File: src/main/webapp/src/components/Sections/PDFLine.tsx (hunk 2 above)
Comment (gwyndol1ne) [reply src/main/webapp/src/components/Sections/PDFLine.tsx]: .

File: src/main/webapp/src/components/Sections/PDFLine.tsx (hunk 3 above)
Comment (gwyndol1ne) [reply src/main/webapp/src/components/Sections/PDFLine.tsx]: .

File: src/main/webapp/src/components/Statistics/Statistics.tsx (hunk 4 above)
Comment (gwyndol1ne) [reply src/main/webapp/src/components/Statistics/Statistics.tsx]: .

File: src/main/webapp/src/components/Statistics/Statistics.tsx (hunk 5 above)
Comment (gwyndol1ne) [reply src/main/webapp/src/components/Statistics/Statistics.tsx]: .

**Review by Darderion — APPROVED** [review]

**Pull request merged**